app/views/historic_appointments/show.html.erb

```diff
@@ -1,57 +1,74 @@
+      breadcrumbs: [
+        {
+          title: "Home",
+          url: "/"
+        },    
+        {
+          title: "History of the UK Government",
+          url: "/government/history"
+        }
+      ]
     } %>
   </div>
-</header>
 
-<%= render partial: 'role_appointments_list', locals: { role: @role } %>
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/title", {
+      context: "Past #{@role.name.pluralize}",
+      title: @person.name,
+    } %>
 
-<%= content_tag_for(:div, @historical_account, class: "block person-info") do %>
-  <div class="name-title">
-    <div class="inner">
-      <h2>
-        <%= @person.name %>
-        <span><%= @historical_account.political_membership %> <%= previous_dates_in_office(@role, @person) %></span>
-      </h2>
-    </div>
+    <%= render "govuk_publishing_components/components/lead_paragraph", {
+      text: "#{@historical_account.political_membership} #{@historical_account.previous_dates_in_office}",
+    } %>
+    <%= render "govuk_publishing_components/components/lead_paragraph", {
+      text: @historical_account.summary,
+    } %>
   </div>
-  <div class="person-profile">
-    <div class="inner">
-      <div class="info">
-        <%= @person.image %>
-        <%= historical_fact('Born', @historical_account.born) %>
-        <%= historical_fact('Died', @historical_account.died) %>
-        <%= historical_fact('Dates in office', previous_dates_in_office(@role, @person)) %>
-        <%= historical_fact('Political party', @historical_account.political_membership) %>
-        <%= historical_fact('Major acts', format_with_html_line_breaks(@historical_account.major_acts)) %>
-        <%= historical_fact('Interesting facts', format_with_html_line_breaks(@historical_account.interesting_facts)) %>
+
+  <div class="govuk-grid-column-two-thirds">
+    <div class="person-profile">
+      <div class="person-profile__image">
+        <%= image_tag @person.image_url, alt: "", loading: "lazy" %>
+      </div>
+      <div class="person-profile__text">
+        Picture of <%= @person.name %>
       </div>
     </div>
+
+    <% @historical_account.appointment_info_array.each do |info| %>
+      <% unless info[:text].blank? %>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: info[:title],
+          font_size: "m",
+          margin_bottom: 1,
+        } %>
+        <p class="govuk-body"><%= info[:text] %></p>
+      <% end %>
+    <% end %>
+
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Biography",
+      font_size: "m",
+      margin_bottom: 1,
+    } %>
+
+    <%= govspeak_to_html @historical_account.body %>
   </div>
-  <div class="person-detail">
-    <div class="inner">
-      <p class="govuk-body-l"><%= @historical_account.summary %></p>
 
-      <%= govspeak_to_html @historical_account.body %>
-    </div>
+  <div class="govuk-grid-column-one-third">
```

Comment:
i'm guessing a design change was to remove the "big numbers" entirely?

Reply:
Yup. In summary:

- With whitehall in it's at-the-time messy link state it was a huge hassle to make it look nice
- Nikin pointed out that the numbers on the past prime ministers page are misleading as they account for multiple terms as PM